devito/types/grid.py

```diff
@@ -907,6 +908,180 @@ def bounds(self):
         return self._local_bounds
 
 
+class Border(SubDomainSet):
```

Comment:
Would call it Boundary

Reply:
A boundary can be internal, whilst a border explicitly goes around the edge of something. Would also likely squat on the namespace of any immersed boundary function down the line